cruise_company_spring/src/main/java/com/example/cruise_company/controller/LinerController.java

```diff
@@ -0,0 +1,62 @@
+package com.example.cruise_company.controller;
+
+import com.example.cruise_company.controller.dto.LinerDto;
+import com.example.cruise_company.service.LinerService;
+import com.example.cruise_company.service.model.Liner;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@Slf4j
+@RequiredArgsConstructor
+public class LinerController {
+
+  private final LinerService linerService;
+
+  @ResponseStatus(HttpStatus.OK)
+  @GetMapping(value = "/liner")
```

Comment:
I would recommend using @RequestMapping("/liner") instead of duplicate the same url in each method